tests/subsys/net/lib/nrf_cloud/nrf_cloud_test/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(nrf_cloud_test)
+set(NRF_SDK_DIR ${ZEPHYR_BASE}/../nrf)
+cmake_path(NORMAL_PATH NRF_SDK_DIR)
+
+# Generate test runner
+test_runner_generate(src/nrf_cloud_test.c)
+
+# NRF CLOUD TEST START
+FILE(GLOB app_sources src/nrf_cloud_test.c)
+target_sources(app PRIVATE ${app_sources})
+
+target_sources(app
+	PRIVATE
+	${NRF_SDK_DIR}/subsys/net/lib/nrf_cloud/src/nrf_cloud.c
+)
+
+zephyr_include_directories(${NRF_SDK_DIR}/subsys/net/lib/nrf_cloud/include/)
+zephyr_include_directories(${ZEPHYR_BASE}/subsys/testsuite/include)
+zephyr_include_directories(${ZEPHYR_BASE}/../modules/lib/cjson)
+
+set_source_files_properties(
```

Comment:
A comment here explaining that these are mocked files that need to not be included from true NCS would be a good idea IMO